resources/core/charts/jaeger/templates/virtualservice.yaml

```diff
@@ -8,14 +7,26 @@ spec:
   hosts:
   - jaeger.{{ .Values.global.domainName }}
   gateways:
-  - {{ .Values.global.istio.gateway.name }}
+  - core-{{ .Chart.Name }}-gateway
```

Comment:
Why can't you use gateway defined for kyma? https://github.com/kyma-project/kyma/blob/master/resources/core/charts/gateway/templates/gateway.yaml

Reply:
We have defined a new “security context” only for Jaeger UI which is independent of the others already existing. Therefore, we have a new “static client” for dex, see the related PR.

Jaeger UI will be used by developers in addition to admins or support. Compromising it should not imply that the others are compromised too.